dist/module.js

```diff
@@ -48,7 +48,7 @@ const moduleDefaults = {
 };
 
 function assignDefaults(strategy, defaults) {
```

Comment:
A bunch of these changes in the dist folder files were autogenerated-- I'm guessing they're all fine considering things still actually work properly

Reply:
Let's reset these files using git, then re-run the build process. That way we ideally will not see the "style" changes like quotes etc in there, just real code updates.